themes/classimax-premium/login.html

```diff
@@ -26,13 +26,7 @@
   <!-- CUSTOM CSS -->
   <link href="css/style.css" rel="stylesheet">
 
-
-  <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
-  <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-  <!--[if lt IE 9]>
-  <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
-  <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
-  <![endif]-->
+  <!-- this is my login page -->

```

Comment:
can you please explain why we did added this comment here?

Reply:
thanks for correction, added this by mistake

Reply:
@Rahi776 Done!